app/components/topbar/Sidebar.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import React, { Component } from 'react';
+import type { Node } from 'react';
+import { observer } from 'mobx-react';
+import SideBarCategory from './SideBarCategory';
+import styles from './Sidebar.scss';
+import type { Category } from '../../config/topbarConfig';
+import classnames from 'classnames';
+
+import yoroiLogo from '../../assets/images/sidebar/yoroi_logo.inline.svg';
+import yoroiLogoExpanded from '../../assets/images/sidebar/yoroi_logo_expanded.inline.svg';
+import walletsIcon from '../../assets/images/sidebar/my_wallets.inline.svg';
+import transferIcon from '../../assets/images/sidebar/transfer_wallets.inline.svg';
+import settingsIcon from '../../assets/images/sidebar/settings.inline.svg';
+import toggleIcon from '../../assets/images/sidebar/open_sidebar.inline.svg';
+
+
+type Props = {|
+  +children?: ?Node,
+  +categories?: Array<Category>,
+  +isActiveCategory?: Function,
```

Comment:
In multiple places you use the type `Function`. Please avoid using this when possible and prefer things like `Category => boolean`